Bootstrap/Update.php

```diff
@@ -223,4 +224,18 @@ private function clearConfigCache()
             $cacheManager->clearConfigCache();
         }
     }
+
+    private function updateConnectAttribute(){
+        if (version_compare($this->version, '1.0.6', '<=')) {
+            $result = Shopware()->Db()->query("SELECT value FROM s_plugin_connect_config WHERE name = 'connectAttribute'");
+            $row = $result->fetch();
+            if ($row) {
+                Shopware()->Db()->query("
+                    UPDATE `s_articles_attributes` 
+                    SET `connect_reference` = `attr" . $row['value'] . "` 
```

Comment:
But this way I set connect_reference to the value from the attribute field that is used. By default this will be attr19. Not "attr19" is written into the reference field, but the value of attr19.